### PR TITLE
implement recently seen indicator

### LINKED ISSUE #1674: Recently seen indicator
counterpart of the Android PR https://github.com/deltachat/deltachat-android/pull/2371

The screenshots in the linked PR show in which screens the indicator should appear.

### PR DESCRIPTION
**Pull request by cyBerta**
closes #1674

### CODE REVIEW

File: DcCore/DcCore/Helper/DcUtils.swift [hunk 1]
```diff
@@ -131,4 +131,15 @@ public struct DcUtils {
...
     }
+
+    public static func showRecentlySeen(context: DcContext, chat: DcChat) -> Bool {
+        var recentlySeen = false
+        if !chat.isSelfTalk && !chat.isGroup && !chat.isMailinglist && !chat.isDeviceTalk {
+            let contactIds = chat.getContactIds(context)
+            if contactIds.count == 1 {
+                recentlySeen = context.getContact(id: contactIds[0]).wasSeenRecently
+            }
+        }
```
Comment (r10s) [inline DcCore/DcCore/Helper/DcUtils.swift:142]: on android, we discussed a bit how [get rid of the two additional database calls](https://github.com/deltachat/deltachat-android/pull/2371#discussion_r953833784) - and finally did by caching already loaded values.

however, i'd say unless we encounter some slowness or it is really easy to improve, we can go for it as is.

**Review by r10s — APPROVED** [review]
lgtm

**Comment by Simon-Laux** [comment]
why on the left side instead of right of the avatar? or is this an old screenshot?

**Comment by r10s** [comment]
> why on the left side instead of right of the avatar? or is this an old screenshot?

right is the verified checkmark, see https://github.com/deltachat/deltachat-ios/issues/1674#issuecomment-1230199609

**Pull request merged**